### PR TITLE
Vox Detective Sprites

### PR DESCRIPTION
**Pull request by zxpr0jk**
Electric Boogaloo Edition

vox_ducktective

Removed a redundant instance of cigar, then made cigar, det hat, and hard-worn suit recognize vox anatomy. Added vox specific sprites for such.

### CODE REVIEW

**Comment by zxpr0jk** [comment]
inb4 "oh god what happened to-"

**Comment by ghost** [comment]
More like it

File: code/game/objects/items/weapons/cigs_lighters.dm [hunk 1]
```diff
@@ -409,6 +409,7 @@ MATCHBOXES ARE ALSO IN FANCY.DM
 	item_state = "cigar"
 	smoketime = 1500
 	chem_volume = 20
+	species_fit = list(VOX_SHAPED)
```
Comment (ComicIronic) [inline code/game/objects/items/weapons/cigs_lighters.dm:412]: Will this not stop humans smoking them? I believe you have to use list("include", VOX_SHAPED), or maybe there's a define like INCLUDE.

Comment (zxpr0jk) [inline code/game/objects/items/weapons/cigs_lighters.dm:412]: Didn't stop me when I tried it. Joined as human, spawned a cigar, and it loaded the human cigar sprite in my mouth. Lit. Still used human sprite. That's actually the line that makes a lot of things recognize the species/vox folder.

Comment (ihadtoregisterforthis) [inline code/game/objects/items/weapons/cigs_lighters.dm:412]: species_fit seems to determine if the clothing has a different sprite for other races.
You must be thinking of species_allowed

**Comment by Kurfursten** [comment]
Can Hard-Worn Suit carry an N2 tank?

**Comment by zxpr0jk** [comment]
Don't think so. I think the coat can hold an emergency tank though. I usually get nobreath from a geneticist or advanced lungs from a roboticist. Being slaved to the N2 tank is rarely for long.

**Comment by ghost** [comment]
How do the noir suits look on them?

**Comment by Konater** [comment]
Invisible

**Pull request merged**